.github/workflows/sets_check_parallel.yaml

```diff
@@ -0,0 +1,20 @@
+name: Sets Check (parallel)
+
+on:
+    pull_request: null
+    push:
+        branches:
+            - master
+
+jobs:
+    run_all_sets:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v2
+            -   uses: shivammathur/setup-php@v1
+                with:
+                    php-version: 7.4
+                    coverage: none # disable xdebug, pcov
+            -   run: composer global require hirak/prestissimo
```

Comment:
How big was the performance hit to add this?
It's incosistent now with other jobs and AFAIK it's not worth the extra lines in every single file.

I used it before in Travis and there was no notable difference (at that time)

Reply:
Ah I see.
`composer install` goes down ~20s (from 33s to 10s) BUT installing the plugin takes about the same amount of time. So it would only pay off atm if more packages are added (so that the parallel download pays of more)

==> will remove it to stay consistent

Reply:
Thanks!